mkosi/resources/man/mkosi-addon.1.md

```diff
@@ -0,0 +1,49 @@
+% mkosi-addon(1)
+%
+%
+
+# NAME
+
+mkosi-addon — Build addons for unified kernel images for the current system
+using mkosi
+
+# SYNOPSIS
+
+`mkosi-addon [options…]`
+
+# DESCRIPTION
+
+`mkosi-addon` is a wrapper on top of `mkosi` to simplify the generation of PE
+addons containing customizations for unified kernel images specific to the
+running or local system. Will include entries in `/etc/crypttab` marked with
+`x-initrd.attach`, `/etc/kernel/cmdline`, kernel modules and firmwares for the
+running hardware.
```

Comment:
The "kernel modules and firmwares for the running hardware" should be dropped as that now only happens if users add `KernelModulesIncludeHost=` to their local configuration.

Maybe document that if users want to add kernel modules in use on their host system, they can add `KernelModulesIncludeHost=` to their local configuration.